PublicDateAtUSN: 2020-01-20
Candidate: CVE-2019-15796
CRD: 2020-01-20
PublicDate: 2020-01-20
References: 
 https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-15796
 https://usn.ubuntu.com/usn/usn-4247-1
Description: 
 can download from untrusted repositories, contrary to apt behaviour
Ubuntu-Description: 
Notes: 
 mdeslaur> this requires a fix in aptdaemon also
Mitigation: 
Bugs: 
 https://bugs.launchpad.net/ubuntu/+source/python-apt/+bug/1858973
Priority: medium
Discovered-by:
Assigned-to: mdeslaur


Patches_python-apt:
upstream_python-apt: needs-triage
precise/esm_python-apt: needed
trusty_python-apt: ignored (out of standard support)
trusty/esm_python-apt: needed
xenial_python-apt: released (1.1.0~beta1ubuntu0.16.04.7)
bionic_python-apt: released (1.6.5ubuntu0.1)
disco_python-apt: released (1.8.5~ubuntu0.2)
eoan_python-apt: released (1.9.0ubuntu1.2)
devel_python-apt: pending (1.9.4)
